modules/executors/variables.tf

```
@@ -167,3 +167,9 @@ variable "assign_public_ip" {
   default     = true
   description = "If false, no public IP will be associated with the executors."
 }
+
+variable "labels" {
+  type        = map(string)
+  description = "A map of labels to add to all resources"
```

```suggestion
  description = "A map of labels to add to the VMs"
```

Maybe? That is the only thing we apply the labels to it seems like.